packages/mdc-select/README.md

```diff
@@ -269,6 +269,23 @@ If you'd like to maintain the width or height outside of the attribute, you'll n
 
 It is advised that dividers also set `role="presentation"` to disable selection and not cloud accessibility.
 
+### Sass Mixins
+
+To customize the colors of any part of the select, use the following mixins. We recommend you use 
+these mixins within CSS selectors like `.foo-select` to apply styling. 
+
+Mixin | Description
+--- | ---
+`mdc-select-ink-color($color)` | Customizes the text entered into the select.* 
+`mdc-select-fill-color($color)` | Customizes the background color of the select. 
+`mdc-select-label-color($color)` | Customizes the label color of the select in the unfocused state. 
+`mdc-select-focused-label-color($color, $opacity: 0.87)` | Customizes the label color of the select when focused. Changing opacity for the label when floating is optional.
+`mdc-select-bottom-line-color($color)` | Customizes the color of the default bottom line of the select.
+`mdc-select-focused-bottom-line-color($color)` | Customizes the color of the bottom line of the select when focused.
+`mdc-select-selected-text-color($color)` | Customizes the color of the option that has been selected. 
+
+>_NOTE_: * - This style applies to the css only version.
```

Comment:
this note seems unfinished?

Reply:
It references the asterisk on line 279, the mdc-select-ink-color is only applied on the css only version. I'll put it in the description column.